Exdi/exdigdbsrv/ExdiGdbSrv/stdafx.cpp

```diff
@@ -0,0 +1,5 @@
+// stdafx.cpp : source file that includes just the standard includes
+// ExdiGdbSrvSample.pch will be the pre-compiled header

```

Comment:
Old file name.

Reply:
do you mean removing the stdafx.cpp file and remove it from the project as well?

Reply:
fixed